### PR TITLE
Add support for the next release (3.4.0) and improved UI

### PR DESCRIPTION
**Pull request by jonasraoni**

### CODE REVIEW

File: locale/en_US/locale.po [hunk 1]
```diff
@@ -22,37 +22,16 @@ msgstr ""
...
-msgid "plugins.generic.customLocale.localeKeyValue"
-msgstr "Value"
+msgstr "No locales found."
 
 msgid "plugins.generic.customLocale.customize"
 msgstr "Customize"
 
-msgid "grid.action.printChanges"
+msgid "plugins.generic.customLocale.printChanges"
 msgstr "Download a file that lists all the customizations you made."
```
Comment (asmecher) [inline locale/en_US/locale.po:35]: This looks weird in the plugin list:
image

I would recommend "Download changes" or something brief like that.

**Comment by asmecher** [comment]
Looks good, @jonasraoni, and the tests pass! I just made one suggestion above, and otherwise this is ready to merge.

**Comment by jonasraoni** [comment]
Due to the [upgrade process I've introduced](https://github.com/pkp/customLocale/pull/16/commits/5436ae3a34710067e25aa9c173d7becd67be1a87), I got desire to work on the https://github.com/pkp/pkp-lib/issues/7764 before merging this.

But well, I'll just leave a reminder in the other issue to come back here later.

**Comment by asmecher** [comment]
Merged -- thanks, @jonasraoni!

**Pull request merged**